### DISCUSSION TITLE
No focus rectangles on toolbar items

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1223] [Team Posts]
Hi @kybos, hope you're still with us.

Have you any idea why focus rectangles are not being drawn, with custom themes (dark flat),
 on items in the toolbars such as the song length, pitch, tempo, positional boxes etc?

They are drawn if the standard light theme is selected.

I looked in the theme files but could not see anything that alters the focus appearance.

Thanks.

**Comment by kybos** [comment]
I added it to both the new themes, it's now in master.
